### PR TITLE
feat(TFD-4617) : rewrite normalize component for tacokit

### PR DESCRIPTION
**Pull request by amarouni**
- Rename the component to Flatten
- Use the same tcompv0 runtime

### CODE REVIEW

File: processing-beam/src/main/java/org/talend/components/processing/flatten/Flatten.java [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+import org.talend.sdk.component.api.processor.OutputEmitter;
+import org.talend.sdk.component.api.processor.Processor;
+
+import javax.json.JsonObject;
+
+import static org.talend.sdk.component.api.component.Icon.IconType.NORMALIZE;
+
+@Version
+@Processor(name = "Flatten")
+@Icon(custom = "Flatten", value = NORMALIZE)
```
Comment (pbailly) [inline processing-beam/src/main/java/org/talend/components/processing/flatten/Flatten.java]:
```suggestion
@Icon(NORMALIZE)
```

File: processing-beam/src/main/java/org/talend/components/processing/flatten/Flatten.java [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+@Documentation("Flatten")
```
Comment (pbailly) [inline processing-beam/src/main/java/org/talend/components/processing/flatten/Flatten.java]:
```suggestion
@Documentation("This component allow the user to create multiple records from one field of the incoming data.")
```

File: processing-beam/src/main/java/org/talend/components/processing/flatten/FlattenConfiguration.java [hunk 3]
```diff
@@ -0,0 +1,75 @@
...
+import org.talend.sdk.component.api.configuration.constraint.Required;
+import org.talend.sdk.component.api.configuration.ui.OptionsOrder;
+import org.talend.sdk.component.api.meta.Documentation;
+
+import java.io.Serializable;
+
+import static org.talend.sdk.component.api.component.Icon.IconType.NORMALIZE;
+
+@Data
+@Icon(custom = "Flatten", value = NORMALIZE)
```
Comment (pbailly) [inline processing-beam/src/main/java/org/talend/components/processing/flatten/FlattenConfiguration.java]: The icon is useless on this class
```suggestion
```

File: processing-beam/src/test/java/org/talend/components/processing/flatten/FalttenUtilsTest.java [hunk 4]
```diff
@@ -0,0 +1,564 @@
...
+import org.apache.avro.generic.GenericRecordBuilder;
+import org.junit.Assert;
+import org.junit.Test;
+import org.talend.daikon.exception.TalendRuntimeException;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+
+public class FalttenUtilsTest {
```
Comment (pbailly) [inline processing-beam/src/test/java/org/talend/components/processing/flatten/FalttenUtilsTest.java]:
```suggestion
public class FlattenUtilsTest {
```

File: component-registry.properties [hunk 5]
```diff
@@ -0,0 +1 @@
+localio=org.talend.components:localio:1.0.0-SNAPSHOT
```
Comment (RyanSkraba) [inline component-registry.properties]: Whas this deliberate?  What does it mean?

File: processing-beam/src/main/java/org/talend/components/processing/flatten/Flatten.java [hunk 6]
```diff
@@ -0,0 +1,44 @@
...
+import org.talend.sdk.component.api.processor.Processor;
+
+import javax.json.JsonObject;
+
+import static org.talend.sdk.component.api.component.Icon.IconType.NORMALIZE;
+
+@Version
+@Processor(name = "Flatten")
+@Icon(NORMALIZE)
+@Documentation("This component allow the user to create multiple records from one field of the incoming data.")
```
Comment (RyanSkraba) [inline processing-beam/src/main/java/org/talend/components/processing/flatten/Flatten.java]:
```suggestion
@Documentation("Create multiple records from one field of the incoming record.")
```

**Pull request merged**